tests/unit/components/sl-chart-test.js

```diff
@@ -0,0 +1,121 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+var object = {
```

Comment:
Not a fan of the use of `object` as it is not descriptive enough and is similar to a JavaScript concept.  See https://github.com/softlayer/ember-style-guide/blob/master/javascript.md#naming-conventions

`series` was used for the series data so `options` seems acceptable, or `testOptions`.  If select the latter or a differing pattern, then also align `series` to be similar.